### DISCUSSION TITLE
How to override the retries limit?

### DISCUSSION THREAD

**Discussion by EngineerKhan** [apify/crawlee-python#523] [Q&A] [answered]
Is there a way of overriding the [`max_retries`](https://github.com/apify/crawlee-python/blob/master/src/crawlee/_utils/wait.py) while running the crawler?

**Comment by vdusek** [comment, marked as answer]
The `_utils.wait.wait_for` function is an internal utility. It is not intended for direct usage & configuration by users. Could you share your use case or what you're trying to achieve with it?

In case you want to limit the max request retries of your crawler, you can use the `max_request_retries` option:

```python
crawler = PlaywrightCrawler(max_request_retries=5)
```

**Reply by EngineerKhan** [reply]
Thank you. Is there also a similar way of setting the minimum request retries?

**Reply by vdusek** [reply]
No, and I'm not sure if such an option makes sense.
